cppapi/server/tango.h

```diff
@@ -45,6 +45,20 @@
 
 #include <idl/tango.h>
 
+//
+// A short inline function to hide the CORBA::string_dup function
+//
+namespace Tango {
+inline char *string_dup(const std::string &s)
+{ return CORBA::string_dup(s.c_str()); }
+
+	inline char *string_dup(char *s) { return CORBA::string_dup(s); }
```

Comment:
This overload is not needed. You can always use "const char *" even for "char *".

Reply:
@t-b, Thanks!